schema/resolvers.js

```diff
@@ -15,6 +15,13 @@ const assertMultipleChoiceAnswer = answer => {
   }
 };
 
+const getRoutingConditionsFields = async (conditionId, field, ctx) => {
+  const conditionValues = await ctx.repositories.Routing.findAllRoutingConditionValues(
+    { conditionId }
+  );
+  return first(conditionValues.map(conditionValue => conditionValue[field]));
```

Comment:
Why are we mapping the whole array - can't we just pull off the field from the first result.

`conditionValues[0][field]`